common-lib/test/cmr/common/test/util.clj

```diff
@@ -44,20 +60,42 @@
 (deftest are2-test
   (testing "Normal use"
     (util/are2
-      [x y] (= x y)
-      "The most basic case with 1"
-      2 (+ 1 1)
-      "A more complicated test"
-      4 (* 2 2))))
+     [x y] (= x y)
+     "The most basic case with 1"
+     2 (+ 1 1)
+     "A more complicated test"
+     4 (* 2 2)))
+  (testing "Empty call"
+    (util/are2
```

Comment:
are2 is not to be used, please move to are3

Reply:
Removed the new are2 test that I added. Replaced with a new test for a different util function